plugins/rapid7_insightidr/komand_rapid7_insightidr/actions/advanced_query_on_log/action.py

```diff
@@ -39,7 +40,17 @@ def run(self, params={}):
                 data=f"\nTime From: {time_from}\nTime To:{time_to}",
             )
 
-        log_id = self.get_log_id(log_name)
+        if log_id is None and log_name is None:
+            raise PluginException(
+                cause="No values were provided for either log id and log name",
```

Comment:
Suggestion reword: No values were provided for log id or log name

Reply:
The logger message has been updated